pytorch_lightning/metrics/sklearns.py

```diff
@@ -232,6 +236,57 @@ def forward(
                                sample_weight=sample_weight)
 
 
+class BalancedAccuracy(SklearnMetric):
+    """ Compute the balanced accuracy score
+
+    Warning:
+        Every metric call will cause a GPU synchronization, which may slow down your code
+
+    Example:
+
+        >>> y_pred = torch.tensor([0, 0, 0, 1])
+        >>> y_true = torch.tensor([0, 0, 1, 1])
+        >>> metric = BalancedAccuracy()
+        >>> metric(y_pred, y_true)
+        tensor([0.8333])
+
+    """
+
+    def __init__(
+            self,
+            adjusted: bool = False,
+            reduce_group: Any = group.WORLD,
+            reduce_op: Any = ReduceOp.SUM):
+        """
+        Args:
+            adjusted: If ``True``, the result sis adjusted for chance, such that random performance
+                corresponds to 0 and perfect performance corresponds to 1
+            reduce_group: the process group for DDP reduces (only needed for DDP training).
+                Defaults to all processes (world)
+            reduce_op: the operation to perform during reduction within DDP (only needed for DDP training).
+                Defaults to sum.
+        """
+        super().__init__('balanced_accuracy_score',
+                         reduce_group=reduce_group,
+                         reduce_op=reduce_op,
+                         adjusted=adjusted)
+
+    def forward(self, y_true: np.ndarray, y_pred: np.ndarray,
```

Comment:
@SkafteNicki can we switch y_true and y_pred here? I've done that to be consistent with PyTorch notation where its first red and then target.

Reply:
Of cause :)